### PR TITLE
Quick fix for #184

### LINKED ISSUE #184: Output files are not BIDS compliant
## Expected Behavior

Following [BIDS 1.2.2](https://bids-specification.readthedocs.io/en/v1.2.2/), the `.tsv.gz` files should have a column for each channel.

## Actual Behavior

Our output has a row for each channel

## Possible solution

We can solve this in two ways:
1. a very easy very quick `BlueprintOutput.timeseries` transpose before (during) the final export to `tsv`, or
2. a less easy, less quick inversion of `BlueprintOutput.timeseries` indexes, followed by adaptation of the scripts, so that `BlueprintOutput.timeseries` will be always created and managed as if we were importing a BIDS compliant `tsv` file.

I prefer option number 2, that could be a bit less easy, but will work very well in case we need to import files or we need this object to be used in other libraries that only import BIDS compliant files. But I'd like to know what y'all think (starting with code contributors @vinferrer @rmarkello @eurunuela)

(thanks to @sangfrois for pointing it out)

### PR DESCRIPTION
**Pull request by smoia**
Closes #184 

## Proposed Changes

  - timeseries matrix gets transposed during its export, to be BIDS compliant.

(thanks to @sangfrois for pointing it out)

### CODE REVIEW

File: phys2bids/phys2bids.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
                 outfile = f'{outfile}_{uniq_freq}'
 
         LGR.info(f'Exporting files for freq {uniq_freq}')
-        savetxt(outfile + '.tsv.gz', phys_out[uniq_freq].timeseries,
+        savetxt(outfile + '.tsv.gz', transpose(phys_out[uniq_freq].timeseries),
```
Comment (rmarkello) [inline phys2bids/phys2bids.py]:
```suggestion
        savetxt(outfile + '.tsv.gz', phys_out[uniq_freq].timeseries.T,
```

File: phys2bids/phys2bids.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 import datetime
 
 from copy import deepcopy
-from numpy import savetxt
+from numpy import savetxt, transpose
```
Comment (rmarkello) [inline phys2bids/phys2bids.py]:
```suggestion
from numpy import savetxt
```

**Comment by smoia** [comment]
Sure thing!

**Comment by smoia** [comment]
@rmarkello when you agree, you can merge it in!

**Pull request merged**